### DISCUSSION TITLE
Accessing i18n configuration from localeDetector

### DISCUSSION THREAD

**Discussion by lewebsimple** [nuxt-modules/i18n#2875] [Q&A]
Hello,

When using a path strategy which uses prefixes, the locale detector should be able to determine the current locale from the path prefix using `new URL(event.headers.get("Referer") || "")`

The only problem is accessing the i18n configuration from `defineI18nLocaleDetector` ... is this possible ?

**Comment by DavidDeSloovere** [comment]
There is only access to `{ defaultLocale: Locale; fallbackLocale: FallbackLocale; }`
I would also like to get acceess to the full config, especially to validate against the configured locales.
Maybe you can create an issue?
